areas-of-d-usage.dd

```diff
@@ -154,40 +155,39 @@ $(AREA_SECTION3 $(LNAME2 operating_systems, Operating systems), $(ARTICLE_FA_ICO
 
 $(AREA_SECTION3 $(LNAME2 embedded, Embedded applications), $(ARTICLE_FA_ICON gears),
 
-    Using $(HTTPS github.com/ldc-developers/ldc, LDC) (with LLVM as backend)
-    enables one to target many CPU architectures,
-    ranging from ARM and MIPS-based embedded systems and smartphones
-    over server systems based on POWER and Sparc up to the "big iron" z System.
+    Using $(HTTPS github.com/ldc-developers/ldc, LDC) (the LLVM D compiler)
+    enables targeting most popular CPU architectures,
+    starting from ARM and MIPS-based embedded systems and smartphones,
+    ranging to server systems based on POWER and Sparc, and up to the "big iron" z Systems.
 
 ))
 
 $(SECTION2 $(LNAME2 cutting-edge, Cutting-edge research), $(SECTION_FA_ICON space-shuttle)
 
 $(DIVC area-section2,
-    D is similar to popular scientific languages like Python as it provides
-    the conveniences of a modern language with a friendly syntax and is
-    garbage collected by default, allowing you to worry about writing your program,
-    rather than handling memory issues that few scientific programmers need to worry about.
-    However it is is a compiled, statically typed language that can produce code as fast as C,
-    allows to easily parallelize your algorithm and pipelines,
-    and as a system language it gives complete control (including manual memory management)
-    if you need to squeeze the last few cycles out of your processor to make your program
-    even faster.
-
-    We now spotlight a few specific areas:
+    D is similar to popular scientific languages like Python, as it provides
+    the conveniences of a modern language with a friendly syntax, and is
+    garbage collected by default. Convenience features allow spending time on rapid prototyping,
+    rather than fighting memory issues that few scientific programmers need to worry about.
+    However, it is also a compiled, statically typed language that can produce code as fast as C,
+    allows easily parallelizing algorithms and pipelines,
+    and, as a system language, provides complete control (including manual memory management)
+    to get those last few extra cycles.
+
+    A few specific spotlight areas:
```

Comment:
Hmm it's not that popular, see e.g. Google:
https://www.google.com/search?q=spotlight+area&q=%22spotlight+area%22